internal/backend/runtime/omni/pkg/check/etcd.go

```diff
@@ -97,7 +97,7 @@ func Etcd(ctx context.Context, r controller.Reader, clusterName string) error {
 		return newError(
```

Comment:
I think the condition `(len(members)-1)%2 != 0` is wrong, and we should re-do the check (and probably messaging):

* n = 1 => OK (while it can't tolerate any loss)
* n = 2 => NOK 
* n = 3 => OK
* n = 4 => NOK (while it can tolerate a single member loss still, quorum is 3)
* n = 5 => OK

Reply:
I don't think the condition is wrong. Message might be not accurate though.

Reply:
updated the message